docs/products/access/topics/custom-registration/custom-registration-v1.md

```diff
@@ -0,0 +1,132 @@
+
+| Parameter | Description                  |
+|-----------|------------------------------|
+| `idp`     | Identity provider identifier |
+
+JSON body parameters:
+
+| Param            | Required                                | Description                                                                        |
+|------------------|-----------------------------------------|------------------------------------------------------------------------------------|
+| `transaction_id` | yes (`TWO_STEP`) otherwise optional     | Generated in **Init step**. For `TWO_STEP`, ensures same transaction               |
+| `profile_id`     | yes (dynamic client) otherwise optional | The profile ID of the user on the Onegini SDK, for static clients it can be omited |
+| `data`           | no                                      | Raw registration request data which will be provided to the Extension Engine       |
+| `scope`          | no                                      | An array of scopes. If none are specified the default scopes are granted.          |
+
+Example request:
+```http
+POST /oauth/custom-registration/example-custom-registration-idp/complete HTTP/1.1
+Host: onegini.example.com
+Content-Type: application/json
+Authentication: Basic Y2xpZW50OnNlY3JldA==
+
+{
+  "transaction_id": "123123",
+  "profile_id": "123efg"
+  "data": "{\"custom_json_key\":\"custom_ json data\"}",      //optional, e.g. challenge code response
+  "scope": ["read", "write"],
+}
+```
+
+Example successful response
+```http
+HTTP/1.1 200 Ok
+Content-Type: application/json;charset=UTF-8
+Cache-Control: no-store
+Pragma: no-cache
+
+{
+    "status": 2000,
+    "oauth_token": {
+        "token_type": "bearer",
+        "access_token": "8A5AB83A3C6B7AAC41471C1205167A35E0F9281ED277EE2FDE6E8DE30972936D",
+        "refresh_token": "8CAE26B2B8E8EC18B4D432886448C7F99B558063C517BA41F30966B37C104983",
+        "expires_in": 3600,
+        "profile_id": "abc123"
+    },
+    "data": "{\"custom_json_key\":\"custom json data\"}", // optional
+}
+```
+
+In the event of an error in the Access Service, one of the following [error codes](#token-server-error-codes) will be returned.
+
+Additional error code for this endpoint:
+
+| Status code | Error code      | Description                                           |
+|-------------|-----------------|-------------------------------------------------------|
+| 400         | invalid_scope   | The requested scope is invalid, unknown or malformed. |
+| 400         | invalid_profile | The specified profile is invalid.                     |
+
+It is up to the script's execution in the XE to determine if the request was successful or not when everything looks fine for the Access Service. For all these
+scenarios, a `200 OK` JSON response returned to the SDK which contains:
+
+| Param           | Description                                                                                               |
```

Comment:
The example response contains more fields than mentioned in this table. Would be good to align those. 
`refresh_token` is only returned when refresh tokens are enabled for this client.
Do we return an `id_token` as well when you add the scope `openid`?